src/Illuminate/Cache/Lock.php

```diff
@@ -51,6 +65,13 @@ public function __construct($name, $seconds)
      */
     abstract public function release();
 
+    /**
+     * Returns the owner value written into the driver for this lock.
+     *
+     * @return mixed
```

Comment:
We probably want this to be `string|null`?

Reply:
@GrahamCampbell any opinion on that one?

Reply:
Yes, if that is their type.